tests/scripts/customlock-keepunlocked-keys.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# This script must be run within a devel-su -p shell
+# The daemon must be running in autotest mode (with --test option)
+
+# Create a collection and test key store/use/delete
+echo "Creating customlock keep-unlocked collection MyCollection, user passphrase required..."
+secrets-tool --test --create-collection --keep-unlocked org.sailfishos.secrets.plugin.encryptedstorage.sqlcipher.test MyCollection
```

Comment:
Use environment variables instead of hardcoding `MyCollection` and the plugin name.